scripts/run-pg-dev.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+export PG_CONTAINER_NAME="postgres"
+PG_IMAGE_NAME="postgres:9.5"
+PG_HOST_DATA="/tmp/pgdata"
```

Comment:
I think we can simply use default location for a volume (i.e., `-v ${PG_CONTAINER_DATA}`). `/tmp` sometimes is a RAM disk.